classes/PodsAPI.php

```diff
@@ -2930,13 +2930,13 @@ public function save_field( $params, $table_operation = true, $sanitized = false
 				 *      @type null|array  $old_options The old field options (if preexisting).
 				 * }
 				 */
-				do_action( 'pods_api_save_field_table_pre_alter', $definition_mode, $pod, $field['type'], $field, [
+				do_action( 'pods_api_save_field_table_pre_alter', $definition_mode, $pod, $field['type'], $field, array(
 					'simple'         => $simple,
 					'definition'     => $definition,
 					'old_name'       => $old_name,
 					'old_definition' => $old_definition,
 					'old_options'    => $old_options,
-				] );
+				) );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 2ec12c2ca399c98cc82b7c1903d5ef8c</sub>